packages/create-plugin/templates/github/ci/.github/workflows/release.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GRAFANA_API_KEY: $\{{ secrets.GRAFANA_API_KEY }} # Requires a Grafana API key from Grafana.com.
+      GRAFANA_ACCESS_POLICY_TOKEN: $\{{ secrets.GRAFANA_ACCESS_POLICY_TOKEN }}
```

Comment:
this template will generate plugins with both variable environments but we probably want them to start using `GRAFANA_API_KEY`. What about leaving the `GRAFANA_API_KEY` line commented and adding a comment on top saying that it is "deprecated"?

Reply:
As discussed on slack please update the step further down that warns about a missing `GRAFANA_API_KEY`.